pynamodb/models.py

```diff
@@ -886,38 +892,34 @@ def _get_indexes(cls):
         """
```

Comment:
Any reason you didn't move the call to `_get_indexes()` in `create_table` over to `cls._indexes`? Not really worried about performance of that, but just for consistency

Reply:
cls._indexes no longer returns the index_data (it's not cached at all since it is only used in create table)

Reply:
Whoops, missed that. Thanks